modules/commons.py

```diff
@@ -161,25 +160,19 @@ def file_content(file_path, debug, container_name):
                     content = []
                     for line in file.readlines():
                         content.append(line[:constants.END])
-
             except PermissionError:
                 cat_file_command = f'sudo cat {file_path}'
                 pipe_cat_file = run_command.command_output(cat_file_command, debug, container_name)
                 content = pipe_cat_file.stdout
                 if content:
                     content = content.split('\n')[:constants.END]
-    if content:
-        print(constants.FULL_NEUTRAL_RESULT_MESSAGE.format('Yes'))
-        print(constants.FULL_EXPLANATION_MESSAGE.format(f'The {file_path} exists in your system'))
-    else:
-        print(constants.FULL_NEUTRAL_RESULT_MESSAGE.format('No'))
-        print(constants.FULL_EXPLANATION_MESSAGE.format(f'The {file_path} does not exist in your system'))
     return content
 
 
 def re_start_of_version(full_version):
     """Returns the start of a version using regex."""
-    return re.search(r'\d*\.\d*', full_version).group()
+    if re.search(r'\d*\.\d*', full_version):
```

Comment:
There is no need to run the regex twice.
You can put it in a variable and then check its value the second time.